app/views/generic_files/_browse_everything.html.erb

```diff
@@ -1,30 +1,2 @@
-<%= render partial: 'generic_files/upload/agreement' %>
-<div class="alert alert-success">
-  <%=  t("sufia.upload.cloud_timeout_message", contact_href: link_to(t("sufia.upload.alert.contact_href_text"), sufia.contact_form_index_path)) %>
+<div >
```

Comment:
This and the other empty view are here to override the views that are provided by sufia